src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/commands/mobile/get_interior_vehicle_data_request.cc

```diff
@@ -144,14 +229,30 @@ void GetInteriorVehicleDataRequest::on_event(
   }
 
   if (result) {
+    app_mngr::ApplicationSharedPtr app =
+        application_manager_.application(connection_key());
+
+    DCHECK_OR_RETURN_VOID(app);
+    if (TheLastAppShouldBeUnsubscribed(app)) {
+      interior_data_cache_.ClearCache();
```

Comment:
@LuxoftAKutsan I think that we should clear cache for received module type, not all cache

Reply:
@LitvinenkoIra @yang1070  fixed a65946326e565d123ee9833420a572323520421e